packages/agent/src/prompt/prompts.ts

```
@@ -1,5 +1,6 @@
-import { MessageContent } from '@langchain/core/messages';
+import { BaseMessage, MessageContent } from '@langchain/core/messages';
```

ESLint

[error] 'BaseMessage' is defined but never used.

(@typescript-eslint/no-unused-vars)